set.mm

```diff
@@ -462181,6 +462245,18 @@ its graph has a given second element (that is, function value).
       WJXAWRFOWPEWKWLWMWQFOWNWO $.
   $}
 
+  ${
+    subne0nn.1 $e |- ( ph -> M e. CC ) $.
+    subne0nn.2 $e |- ( ph -> N e. CC ) $.
+    subne0nn.3 $e |- ( ph -> ( M - N ) e. NN0 ) $.
+    subne0nn.4 $e |- ( ph -> M =/= N ) $.
+    $( A nonnegative difference is an integer if the two numbers are not equal.
```

Comment:
Maybe better to say "A nonnegative difference is positive if the two numbers are not equal."?

Reply:
Changed in upcoming commit.